gc/base/MemoryPoolAddressOrderedList.hpp

```diff
@@ -63,15 +63,25 @@ class MM_MemoryPoolAddressOrderedList : public MM_MemoryPoolAddressOrderedListBa
 	
 	MM_LargeObjectAllocateStats *_largeObjectCollectorAllocateStats;  /**< Same as _largeObjectAllocateStats except specifically for collector allocates */
 
-	MM_HeapLinkedFreeHeader *_firstUnalignedFreeEntry; /**< it is only for Balanced GC copyforward and non empty survivor region */
-	MM_HeapLinkedFreeHeader *_prevFirstUnalignedFreeEntry;
+	MM_HeapLinkedFreeHeader *_firstCardUnalignedFreeEntry; /**< it is only for Balanced GC copyforward and non empty survivor region */
+	MM_HeapLinkedFreeHeader *_prevCardUnalignedFreeEntry;
+
+	void* _parallelGCAlignmentBase;
+	uintptr_t _parallelGCAlignmentSize;
 protected:
 public:
 	
 /*
  * Function members
  */	
 private:
+	/* Align a TLH to meet boundary restrictions. Certain phases of some GCs may require that TLHs not span heap chunks for parallel processing.*/
+	bool alignTLHForParallelGC(MM_EnvironmentBase *env, MM_HeapLinkedFreeHeader *freeEntry, uintptr_t *consumedSize);
+	uintptr_t determineConsumedSizeForTLH(MM_EnvironmentBase *env, MM_HeapLinkedFreeHeader *freeEntry, uintptr_t maximumSizeInBytesRequired);
```

Comment:
Meanwhile checked, and we seem not to use 'determine' often or at all. A simple 'get' seems to be more common. Pick whatever you like (or if you have a better suggestion?).

Reply:
Agree, get is more suitable.